src/cli/chem-comp-dict/create-table.ts

```diff
@@ -243,8 +243,37 @@ function createAtoms(ccd: DatabaseCollection<CCD_Schema>) {
     );
 }
 
-async function run(out: string, binary = false, ccaOut?: string) {
-    const { bonds, atoms } = await createBonds(!!ccaOut);
+function extractIonNames(ccd: DatabaseCollection<CCD_Schema>) {
+    const ionNames: string[] = [];
+    for (const k in ccd) {
+        const {chem_comp} = ccd[k];
+        if (chem_comp.name.value(0).toUpperCase().includes(' ION')) {
```

Comment:
Are you sure this is a necessary and/or correct condition?

I know this wasn't included in the original list, but should this include anions/cations as well? See for example http://files.rcsb.org/ligands/view/FLC.cif

Also this is PDB, so I would not be surprised if there were names with "-ION" or other variants.

Reply:
Hi David, he based it on an SQL query I used previously to get a list of small ions. Heavy atom count should be below 8 so something like https://www.rcsb.org/ligand/OHX would be the largest.